app/src/routes/api/v1/subscription.router.js

```diff
@@ -249,6 +247,24 @@ class SubscriptionsRouter {
         ctx.body = await StatisticsService.infoByUserSubscriptions(new Date(ctx.query.start), new Date(ctx.query.end), ctx.query.application);
     }
 
+    static async findByIds(ctx) {
+        logger.info(`[SubscriptionsRouter] Getting all subscriptions with ids`, ctx.request.body);
+        if (ctx.request && ctx.request.body && ctx.request.body.ids && ctx.request.body.ids.length > 0) {
+            const subscriptions = await SubscriptionService.getSubscriptionsByIds(ctx.request.body.ids);
+            ctx.body = { data: subscriptions };
```

Comment:
Use the existing serializer if possible

Reply:
Added call to the existing serializer inside the subscription service!